AltStore/Server/Server.swift

```diff
@@ -17,9 +17,9 @@ enum ConnectionError: LocalizedError
     var failureReason: String? {
         switch self
         {
-        case .serverNotFound: return NSLocalizedString("Could not find AltServer.", comment: "")
```

Comment:
Delete this entire file

Reply:
The latest Commit should resolve this issue.